pyairctrl/airctrl.py

```diff
@@ -263,47 +302,53 @@ def main():
     parser.add_argument('--filters', help='read filters status', action='store_true')
     args = parser.parse_args()
 
-    c = AirClient(args.ipaddr)
-    c.load_key()
-    if args.wifi:
-        c.get_wifi()
-        sys.exit(0)
-    if args.firmware:
-        c.get_firmware()
-        sys.exit(0)
-    if args.wifi_ssid or args.wifi_pwd:
-        c.set_wifi(args.wifi_ssid, args.wifi_pwd)
-        sys.exit(0)
-    if args.filters:
-        c.get_filters()
-        sys.exit(0)
+    if args.ipaddr:
+        iplist = [[args.ipaddr]]
+    else:
+        iplist = ssdp(debug=args.debug)
+    
+    for ip in iplist:
```

Comment:
if the user haven't specified an IP and the discovery fails, we should print a message saying that we cannot find the IP

Reply:
```python
        if not iplist:
            print('Air purifier not autodetected. Try --ipaddr option to force specific IP address.')
```
Will it be enough?

Reply:
Yes. Also return exit code 1 in this case.

Reply:
added